.github/workflows/release.yml

```diff
@@ -9,9 +9,17 @@ on:
 permissions:
   # goreleaser writes to the releases api
   contents: write
+  # docker writes packages to container registry
+  packages: write
+
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  IMAGE_NAME: ${{ github.repository }}
 
 jobs:
-  goreleaser:
+  release:
```

Comment:
Can you leave `goreleaser` and create a new `job` for the docker release please?

Reply:
Sure, I've updated the release workflow and added the steps to a `docker-release` job as suggested.